[2025-03-19T19:06:25.618+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-19T19:06:26.184+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_taskflow_Api_v2.get_name scheduled__2024-10-26T00:00:00+00:00 [queued]>
[2025-03-19T19:06:26.265+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_taskflow_Api_v2.get_name scheduled__2024-10-26T00:00:00+00:00 [queued]>
[2025-03-19T19:06:26.272+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-03-19T19:06:26.525+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): get_name> on 2024-10-26 00:00:00+00:00
[2025-03-19T19:06:26.622+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=9692) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-19T19:06:26.625+0000] {standard_task_runner.py:72} INFO - Started process 9715 to run task
[2025-03-19T19:06:26.617+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_taskflow_Api_v2', 'get_name', 'scheduled__2024-10-26T00:00:00+00:00', '--job-id', '2812', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_taskflow_api.py', '--cfg-path', '/tmp/tmpv76921hl']
[2025-03-19T19:06:26.657+0000] {standard_task_runner.py:105} INFO - Job 2812: Subtask get_name
[2025-03-19T19:06:27.250+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_taskflow_Api_v2.get_name scheduled__2024-10-26T00:00:00+00:00 [running]> on host 63fd4b143e0a
[2025-03-19T19:06:28.868+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_with_taskflow_Api_v2' AIRFLOW_CTX_TASK_ID='get_name' AIRFLOW_CTX_EXECUTION_DATE='2024-10-26T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-26T00:00:00+00:00'
[2025-03-19T19:06:28.883+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-19T19:06:28.985+0000] {python.py:240} INFO - Done. Returned value was: {'first_name': 'John', 'last_name': 'Doe'}
[2025-03-19T19:06:29.468+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-19T19:06:29.501+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=dag_with_taskflow_Api_v2, task_id=get_name, run_id=scheduled__2024-10-26T00:00:00+00:00, execution_date=20241026T000000, start_date=20250319T190626, end_date=20250319T190629
[2025-03-19T19:06:30.100+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-19T19:06:30.692+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-19T19:06:30.758+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
